math-libs/CMakeLists.txt

```
@@ -106,6 +105,44 @@ if(THEROCK_ENABLE_PRIM)
   therock_cmake_subproject_provide_package(rocPRIM rocprim lib/cmake/rocprim)
   therock_cmake_subproject_activate(rocPRIM)
 
+  if(THEROCK_BUILD_TESTING)
+    ##############################################################################
+    # rocPRIM tests
+    ##############################################################################
+
+    # rocPRIM_Tests declared as a separate target to stop the tests from clogging up the critical build path for libs
+    # that depend on rocPRIM
+    therock_cmake_subproject_declare(rocPRIM_Tests
+      EXTERNAL_SOURCE_DIR "${THEROCK_ROCM_LIBRARIES_SOURCE_DIR}/projects/rocprim"
+      BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/rocPRIM"
+      BACKGROUND_BUILD
+      CMAKE_ARGS
+        -DHIP_PLATFORM=amd
+        -DROCM_PATH=
+        -DROCM_DIR=
+        -DBUILD_TEST=ON
+        -DUSE_SYSTEM_LIB=ON
+      COMPILER_TOOLCHAIN
+        amd-hip
+      DIR_PREFIX
+        tests_
+      BUILD_DEPS
+        rocm-cmake
+        therock-googletest
+        rocPRIM
+        # rocSPARSE
+        # rocSOLVER
```

remove commented out build deps? Or explain why they are commented out in the code
```suggestion
      BUILD_DEPS
        rocm-cmake
        therock-googletest
        rocPRIM
```